src/wp-includes/theme.php

```diff
@@ -3142,3 +3142,41 @@ function _wp_keep_alive_customize_changeset_dependent_auto_drafts( $new_status,
 		clean_post_cache( $post_id );
 	}
 }
+/**
+ * Get the theme logo through customizer.
+ * Add style with background image if theme logo exists. 
+ * Note: By default, the size of your logo should be no bigger than 80 x 80 pixels 
+ * (though even this can change with custom CSS). 
+ * Also change the link of the <a> link from the blog-url and title attribute from blog description
+ */
+function alter_login_logo() { 
+	$custom_logo_id = get_theme_mod( 'custom_logo' );
+	$image = wp_get_attachment_image_src( $custom_logo_id , 'full' );
+	$url_image = $image[0];
+
+	if (!$url_image) {
+		return;
+	}
+	?>
+    <style type="text/css">
+        #login h1 a, 
+		.login h1 a { 
+			background-image: url(<?php echo $url_image; ?>);
+        }
+    </style>
```

Comment:
If possible, I would rather we modify the login page HTML directly, rather than doing it later in the page cycle by adding a new stylesheet.

Reply:
I have done that in the recent PR update. It takes a different turn of things.